modules/virt-cloning-a-datavolume.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-cloning-a-datavolume-using-smart-cloning.adoc
+
+[id="virt-cloning-a-datavolume_{context}"]
+= Cloning a DataVolume
+
+To clone a DataVolume (DV), do the following:
```

Comment:
Not a huge fan of "do the following". But I think that if you make this a proper procedure module with the .Procedure heading, then you can add a brief lead-in sentence and not have to use this sentence.

Reply:
Cool, I just dropped that phrase then. I've had some editors require it, others hate it. :-)